{"time":"2024-11-09T14:13:40.1787717+07:00","level":"INFO","msg":"using version","core version":"0.18.6"}
{"time":"2024-11-09T14:13:40.1787717+07:00","level":"INFO","msg":"created symlink","path":"d:\\workspace\\leetcode-workspace\\jupyter\\RMIT Hackathon 2024\\wandb\\run-20241109_141340-w57n25bf\\logs\\debug-core.log"}
{"time":"2024-11-09T14:13:40.3031273+07:00","level":"INFO","msg":"created new stream","id":"w57n25bf"}
{"time":"2024-11-09T14:13:40.3031273+07:00","level":"INFO","msg":"stream: started","id":"w57n25bf"}
{"time":"2024-11-09T14:13:40.3031273+07:00","level":"INFO","msg":"writer: Do: started","stream_id":{"value":"w57n25bf"}}
{"time":"2024-11-09T14:13:40.3036399+07:00","level":"INFO","msg":"sender: started","stream_id":"w57n25bf"}
{"time":"2024-11-09T14:13:40.3036399+07:00","level":"INFO","msg":"handler: started","stream_id":{"value":"w57n25bf"}}
{"time":"2024-11-09T14:14:10.3096726+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"}
{"time":"2024-11-09T14:14:33.3322536+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.105:62730->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2024-11-09T14:15:04.4054101+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.105:62764->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2024-11-09T14:15:26.4941922+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": EOF"}
{"time":"2024-11-09T14:16:14.1290082+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"}
{"time":"2024-11-09T14:17:18.4053299+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": context deadline exceeded"}
{"time":"2024-11-09T14:18:48.4064748+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": context deadline exceeded"}
{"time":"2024-11-09T14:20:18.4075247+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": context deadline exceeded"}
{"time":"2024-11-09T14:21:48.4082233+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": context deadline exceeded"}
{"time":"2024-11-09T14:22:48.4125007+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2024-11-09T14:26:13.5326919+07:00","level":"WARN","msg":"sender: taking a long time","seconds":753.2248133,"work":"WorkRecord(*service_go_proto.Record_Run); Control(mailbox_slot:\"k4ccijk8zevj\"  always_send:true  connection_id:\"127.0.0.1:62666\")"}
{"time":"2024-11-09T14:26:43.533269+07:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"}
{"time":"2024-11-09T14:27:44.295244+07:00","level":"INFO","msg":"sender: succeeded after taking longer than expected","seconds":843.9873473,"work":"WorkRecord(*service_go_proto.Record_Run); Control(mailbox_slot:\"k4ccijk8zevj\"  always_send:true  connection_id:\"127.0.0.1:62666\")"}
